GameOfRunes/GameOfRunes/GameEngine/Systems/GestureEntitySystem.swift

```diff
@@ -34,7 +34,7 @@ class GestureEntitySystem: GKComponentSystem<GestureEntityComponent>, System {
     
     func setInitialGesture(for entity: Entity) {
         guard entity.type == .enemyEntity,
-            let enemyNode = entity.component(ofType: SpriteComponent.self)?.node,
+            let enemyNode = entity.component(ofType: SpriteComponent.self)?.node as? SKSpriteNode,
```

Comment:
why do we need to change this to `SKSpriteNode`

Reply:
The `node` is a SKNode, so explicit typecast is required.